Packs/CommonScripts/Scripts/script-DeleteConext.yml

```diff
@@ -43,6 +43,12 @@ script: >
   if (shouldDeleteAll) {
       var keysToKeep = (args.keysToKeep) ? args.keysToKeep.split(',').map(function(item) { return item.trim(); }) : [];
       var keysToKeepObj = {};
+      var DBotScore = false;
```

Comment:
Let's give a better name to this variable

Reply:
done